functions/import-csv.js

```diff
@@ -64,19 +62,23 @@ async function importCsv(req, res) {
 
     file.pipe(csvParser()).on("data", async (row) => {
       try {
-        await insertRow(projectId, layerId, row);
+        inserts.push(insertRow(projectId, layerId, row));
       } catch (err) {
         console.error(err);
-        await res.status(HttpStatus.BAD_REQUEST).end("{}");
-        // TODO(#525): Abort stream on error. How?
+        re.unpipe(busboy);
```

Comment:
Is this supposed to be `res.unpipe()` ? Same question for below

Reply:
Yikes, last minute typo! Fixed.

> Same question for below

What were you referring to? Don't see another instance of `re.`..

Reply:
Oh maybe I was looking at the same one and did not notice, LGTM

Reply:
Thanks for reviewing!